spec/helpers/chat_mirgation_message_spec.rb

```diff
@@ -0,0 +1,25 @@
+RSpec.describe ChatMigrationMessage, type: :helper do
+  describe '#display_message?' do
```

Comment:
the method that is being tested here is :display_chat_migration_message?, rather than :display_message?